### ISSUE TITLE
Add Your Music not working

### ISSUE THREAD

**Issue by rownpown** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#377]
**OS:**Windows 10 64 bit

**Issue Descriptions:** Menu Icon disappearing and Add music from Library doesn't work
![ezgif com-video-to-gif](https://cloud.githubusercontent.com/assets/9003620/13607335/e92edd16-e54f-11e5-8832-b614cb590bbe.gif)

**Steps to Reproduce:** I do not have a subscription to GPM yet, so I cannot confirm if that is why somethings are not showing, but clicking on add music from library no options pop up to allow for music to be added. Additionally, the menu bars icon disappears from the top left.

**Comment by MarshallOfSound** [comment]
It's not supposed to at the moment.  The only reason you can get to the button is because your google play music is a blank slate (I have never tested from a clean account)

Implementing file upload requires spoofing the `chrome://` URI scheme and faking an awful lot of file access API's that are part of chrome, but not part of chromium.  We would have to fake those through the main processes `fs` access

TLDR: Big task, buts its on a TODO list somewhere nearby :+1:

**Comment by MarshallOfSound** [comment]
Unfortunately this means you will have to use the official ways (the music manager or the web version) to upload your music for know

**Comment by rownpown** [comment]
OK! That seems pretty difficult thanks for the answer!!
